app/src/main/java/com/example/firstandroid/MainActivity.kt

```diff
@@ -37,6 +43,7 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
```

Comment:
If we are going to have multiple viewmodels in a class then I would give each a specific name